### ISSUE TITLE
About parameter count problem

### ISSUE THREAD

**Issue by shookwu1210-boop** [Daniellb12/PhaseCoder#1]
Hello, I’m currently reproducing PhaseCoder too. Thanks so much for sharing your code!
When I run python PhaseCoder.py, the output is:

`/usr/local/lib/python3.11/dist-packages/torch/nn/modules/transformer.py:382: UserWarning: enable_nested_tensor is True, but self.use_nested_tensor is False because encoder_layer.norm_first was True
warnings.warn(

PhaseCoder parameters: 2,196,040 (~2.2M)

Spatial embedding shape: torch.Size([4, 256])
Azimuth logits shape:    torch.Size([4, 39])
Elevation logits shape:  torch.Size([4, 19])
Distance logits shape:   torch.Size([4, 14])

Total loss: 7.9064`

The parameter count ~2.2M does not match the ~6M reported in the paper.
